core/controllers/base_test.py

```diff
@@ -200,7 +200,9 @@ def test_redirect_in_logged_out_states(self):
         """Test for a redirect in logged out state on '/'."""
 
         response = self.get_html_response('/', expected_status_int=302)
-        self.assertIn('splash', response.headers['location'])
+        response_path = response.headers['location']
+        self.assertEqual(response_path[-1], '/')
+        self.assertEqual(len(response_path.split('/')), 4)
```

Comment:
Add a comment explaining the significance of "4". Ditto below and for all other such tests.

Reply:
This is changed now, but I've added the comment for the other test.